WEAPON/CreateWeaponObject.md

```diff
@@ -5,26 +5,21 @@ ns: WEAPON
 
 ```c
 // 0x9541D3CF0D398F36 0x62F5987F
-Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
+Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, cs_type(Any) BOOL scriptHostObject);
 ```
 
-```
-Now has 8 params.  
-```
+Create a weapon object that cannot be attached to a ped. If you want to create a weapon object that can be attached to a ped, use [CREATE_OBJECT](#_0x509D5878EB39E842) instead.
 
 ```
-NativeDB Added Parameter 9: Any p8
-NativeDB Added Parameter 10: Any p9
+Now has 8 params.  
 ```
 
 ## Parameters
-* **weaponHash**: 
-* **ammoCount**: 
-* **x**: 
-* **y**: 
-* **z**: 
-* **showWorldModel**: 
-* **scale**: 
-* **p7**: 
-
-## Return value
+* **weaponHash**: The hash of the weapon to create.
+* **ammoCount**: The amount of ammo for the weapon.
+* **x**: X coordinate for the weapon's position.
+* **y**: Y coordinate for the weapon's position.
+* **z**: Z coordinate for the weapon's position.
+* **showWorldModel**:
+* **scale**:
+* **scriptHostObject**: R* always uses `true` for this parameter.
```

Comment:
`customModelHash` (integer) as mentioned up above.

Reply:
Should be good!